src/engine/ruleset/integrations/system/decoders/sysmon-linux.yml

```diff
@@ -258,8 +258,8 @@ normalize:
   - check:
```

Comment:
Could we change this check for this, so that it has the same design as the rest of the checks?

```
 - check:
      - event.code: '4'
```